kitchen.yml

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+# For help on this file's format, see https://kitchen.ci/
+driver:
+  name: docker
+  use_sudo: false
+  privileged: true
+  run_command: /lib/systemd/systemd
+
+# Make sure the platforms listed below match up with
+# the `env.matrix` instances defined in `.travis.yml`
+platforms:
+  - name: debian-9
+    driver_config:
+      image: debian:9
+      provision_command:
+        - apt-get update && apt-get install -y udev locales
+        - echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
+        - locale-gen en_US.UTF-8
+      platform: debian
+
+  - name: debian-8
+    driver_config:
+      image: debian:8
+      provision_command:
+        - apt-get update && apt-get install -y udev locales
+        - echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
+        - locale-gen en_US.UTF-8
+      platform: debian
+
+  - name: ubuntu-18.04
+    driver_config:
+      image: ubuntu:18.04
+      provision_command:
+        - apt-get update && apt-get install -y udev locales && rm -rf /var/lib/apt/lists/*
+        - localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+        - echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
+        - locale-gen en_US.UTF-8
+      platform: ubuntu
+
+  - name: ubuntu-16.04
+    driver_config:
+      image: ubuntu:16.04
+      provision_command:
+        - apt-get update && apt-get install -y udev locales && rm -rf /var/lib/apt/lists/*
+        - localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+        - echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
+        - locale-gen en_US.UTF-8
+      platform: ubuntu
+
+provisioner:
+  name: salt_solo
+  log_level: info
+  salt_version: latest
```

Comment:
Would it be possible to get the pre-salted images?  Or is that for a subsequent PR?

Reply:
I'll do it in a future PR